libafl_bolts/src/shmem.rs

```diff
@@ -921,12 +902,8 @@ pub mod unix_shmem {
                     let map = shmat(id_int, ptr::null(), 0) as *mut c_uchar;
 
                     if map.is_null() || map == ptr::null_mut::<c_uchar>().wrapping_sub(1) {
-                        println!(
-                            "errno : {:?}",
-                            std::io::Error::last_os_error().raw_os_error().unwrap()
-                        );
                         return Err(Error::unknown(format!(
-                            "Failed to map the shared mapping with id {id_int}"
+                            "Failed to map the shared mapping with id {id_int},Last OS error :{:?}",std::io::Error::last_os_error()
```

Comment:
This can't compile, there is no closing quotes here? At least try to compile libafl_bolts (cd libafl_bolts && cargo make)

Reply:
I am really sorry about that , I will check that soon.